setup.cfg

```diff
@@ -5,6 +5,8 @@ max-line-length = 88
 ignore =
     # space before : (needed for how black formats slicing)
     E203,
+    # expected n blank lines
+    E3,
```

Comment:
E3 shows things like "expected 2 blank lines, expected 0" which I think is a bit irrelevant for docstrings - we put the lines one after the other anyway, having empty `>>>` lines in docstrings wouldn't look good